presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -282,7 +283,7 @@ private void printQueryInfo(QueryResults results)
                 reprintLine(String.format("Parallelism: %.1f", parallelism));
             }
 
-            assert terminalWidth >= 75;
+            checkState(terminalWidth >= 75); // otherwise handled above
```

Comment:
Use `verify()` from Guava's `Verify`. This provides a better exception since `IllegalStateException` has a specific definition which isn't met here. More explanation: https://github.com/google/guava/wiki/ConditionalFailuresExplained